db/migrate/20180109102630_clean_up_duplicates_in_containers_tables.rb

```diff
@@ -71,15 +72,16 @@ def cleanup_duplicate_data_delete_all(model, unique_index_columns)
     # Nested tables, not having :ems_id and the foreign_key is a part of the unique index
     ContainerCondition         => %i(container_entity_id container_entity_type name),
     SecurityContext            => %i(resource_id resource_type),
+    Tagging                    => %i(taggable_id taggable_type tag_id),
     ComputerSystem             => %i(managed_entity_id managed_entity_type),
     ContainerEnvVar            => %i(container_id name value field_path),
     ContainerLimitItem         => %i(container_limit_id resource item_type),
     ContainerPortConfig        => %i(container_id ems_ref),
-    ContainerQuotaScope        => %i(container_quota_id scope),
     ContainerQuotaItem         => %i(container_quota_id resource),
+    ContainerQuotaScope        => %i(container_quota_id scope),
     ContainerServicePortConfig => %i(container_service_id name),
     ContainerTemplateParameter => %i(container_template_id name),
-    ContainerVolume            => %i(parent_id parent_type name),
+    ContainerVolume            => %i(parent_id parent_type ems_ref name),
```

Comment:
@cben We use :ems_ref for PersistentVolume and :name for ContainerVolume, is that wanted?